compiler/ras/LimitFile.cpp

```diff
@@ -948,6 +962,8 @@ TR_Debug::print(TR_FilterBST * filter)
          printFilters(filter->subGroup);
          TR_VerboseLog::write("   ]\n");
          }
+      
+      TR_VerboseLog::vlogRelease();
```

Comment:
On line 954 in this file we call `filter->getRegex()->print(false);` which may go to this overload:

```
void SimpleRegex::print(bool negate)
```

However this PR changes the above overload to wrap it in acquire/release statements. This means the verbose log lock will be released after the `print` call on line 954. I'm not exactly sure what happens if we double acquire or double release the lock as we are doing here. @mpirvu perhaps you can confirm?

Reply:
Monitors in OpenJ9 can be acquired recursively. As long as the number of exits matches the number of enters we should be fine